app/packages/grits-net-meteor/client/templates/grits_search.coffee

```diff
@@ -363,13 +363,24 @@ Template.gritsSearch.onRendered ->
     if mode == GritsConstants.MODE_EXPLORE
       _resetSimulationProgress()
       _disableLimit.set(false)
+      async.nextTick( ->
+        if GritsFilterCriteria.departures.get().length isnt 0
+          $("#showThroughput").click()
+      )
       # reset the URL back to the root
       FlowRouter.go('/')
     else
       # initialize the bootstrap slider (this is not rendered by default in grits_search.html)
       # it is done using nextTick to give Blaze template time to render
       async.nextTick(-> $('#simulatedPassengersInputSlider').slider())
       _disableLimit.set(true)
+      # only allow startSimulation to be clicked when an existing simulation
```

Comment:
Did you intend to add this back? It was removed to prevent the simulation from being triggered when the mode is switched.